+exp/ExpTest.m

```diff
@@ -0,0 +1,330 @@
+classdef ExpTest < handle
+%EXPTEST Creates an experiment test panel for testing *Signals* Exp Defs on a personal PC
+%
+% See also: SIGNALSEXPTEST, EUI.MCONTROL
+%
+% Usage:
+% 1) Run the command 'expTestPanel' to launch the 'ExpTestPanel' GUI
+% 2) (Optional) Use the 'Select Subject...' drop-down menu if you want to
+% load a parameter set from a particular subject (see '4)' below) and
+% save a 'block' .mat file
+% 3) Click the 'Select Signals Exp Def' button to choose the *Signals* Exp
+% Def to be run
+% 4) (Optional) Use the 'Select parameters...' drop-down menu if you want
+% to load a specific parameter set
+% 5) (Optional) Click the 'Options' button to select which options you
+% to enable when running the Exp Def:
+%   a) Live-Plotting
+%   b) Save a 'block' file
+% 6) Click the 'Start' button to run the Exp Def. This button will turn
+% into a 'Stop' button after the experiment starts running - click it again
+% to end the experiment (alternatively, press the 'q' keyboard key to
+% stop the experiment). To run another exp def in the same panel (or the
+% same exp def that was just run), repeat 'Usage' from step 3).
+%
+% *Note: While the experiment is running, the position of the mouse
+% cursor will be set to emulate the wheel. Press the 'c' keyboard key
+% while experiment is running to disable/enable this feature.
+%
+% Todo: set properties and methods attributes appropriately
+  
+  %% properties
+  properties %(Access = private)
+    PanelH % 'ExpTestPanel' figure handle
+    MainGrid % main 'uix.GridFlex' object; child of 'PanelH'
+    ExpGrid % top 'uix.Grid' object; child of 'MainGrid'
+    ExpTopBox % 'uix.HBox' object, containing UI elements to run the Exp Def; child of 'ExpGrid'
+    ExpBottomBox % 'uix.HBox' object, containing UI elements which display experiment info; child of 'ExpGrid'
+    SubjectList % 'bui.Selector' object containing the subject-list
+    Subject % string of currently selected subject (optional)
+    SelectExpDef % handle to 'Select Signals Exp Def' push-button
+    ExpDef % function handle to currently chosen Signals Exp Def
+    ExpDefPath % fullfile path of ExpDef
+    OptionsButton % handle to 'Options' push-button
+    LivePlot = 1 % option for live-plotting signals during experiment
+    SaveBlock = 0 % option for saving 'block' file at experiment end
+    StartButton % handle to 'Start' push-button
+    ParamPanel % 'uix.Panel' object containing the 'ParamGrid'; child of 'MainGrid'
+    ParamGrid % 'uix.GridFlex' object containing 'ParamTopBox' and 'ParamBottomBox'; child of 'ParamPanel'
+    ParamTopBox % 'uix.HBox' object containing 'bui.Selector' object for list of saved parameter sets; child of 'ParamGrid'
```

Comment:
Personally I recommend writing these above the properties as they still show up in the documentation and you can also have the text run on to the next line (ctrl+J).  Both are valid of course. https://uk.mathworks.com/help/matlab/matlab_prog/create-help-for-classes.html

Reply:
ok, thanks for the info